actgen/agents/action_dqn.py

```diff
@@ -5,37 +5,37 @@
 
 
 class ActionDQNAgent(DQNAgent):
-    def __init__(self, env, params):
-        super().__init__(env, params)
+    def __init__(self, observation_space, action_space, get_duplicate_actions, params):
+         super().__init__(observation_space, action_space, get_duplicate_actions, params)
 
     def _get_q_targets(self, batch):
         with torch.no_grad():
             # Compute Double-Q targets
-            next_states = torch.stack(batch.next_state, dim=0).float()
+            next_states = torch.stack(batch.next_state, dim=0).float().to(self.params['device'])
             aps = []
```

Comment:
It's not clear what `aps` is. Can you be more descriptive with the name?

Reply:
it's intended to be `a_prime`s, I will change it to `next_actions`